src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/AesCcm.Unix.cs

```diff
@@ -84,9 +84,8 @@ private void ImportKey(ReadOnlySpan<byte> key)
             {
                 Interop.Crypto.CheckValidOpenSslHandle(ctx);
                 Interop.Crypto.EvpCipherSetCcmNonceLength(ctx, nonce.Length);
-                Interop.Crypto.EvpCipherSetCcmTag(ctx, tag);
-
                 Interop.Crypto.EvpCipherSetKeyAndIV(ctx, _key, nonce, Interop.Crypto.EvpCipherDirection.Decrypt);
+                Interop.Crypto.EvpCipherSetCcmTag(ctx, tag);
```

Comment:
Was this a bug? If so, was a test failing?

Reply:
This is from when I initially tried to get the logic for AesCcm to work for both OpenSSL and Android. I'll switch it back.